lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -932,7 +932,17 @@ bool PrimUpdaterManager::canEditAsMaya(const Ufe::Path& path) const
     return updater ? updater->canEditAsMaya() : false;
 }
 
-bool PrimUpdaterManager::discardEdits(const Ufe::Path& pulledPath)
+bool PrimUpdaterManager::discardEdits(const MDagPath& dagPath)
```

Comment:
Orphaned objects (i.e. maya's objects that were created as a result of pull operation, but no longer have a valid prim associated with it) won't be able to provide a prim path. Switching to MDagPath to make it more evident since before we were expecting it always to be a two-segment path.